registry/recipes/nginx.md

```diff
@@ -160,7 +160,9 @@ Review the [requirements](/registry/recipes/index.md#requirements), then follow
 
     ```yaml
     nginx:
-      image: "nginx:1.9"
+      # Note : Only the alpine flavour supports bcrypt
```

Comment:
Add something about "If you want to drop use of bcrypt, you can change the `htpasswd` command to not include the `-B` argument".